code/modules/movespeed/modifiers/mobs.dm

```diff
@@ -97,12 +97,27 @@
 	variable = TRUE
 	flags = IGNORE_NOSLOW
 
-/datum/movespeed_modifier/tarantula_web
-	multiplicative_slowdown = 5
+/datum/movespeed_modifier/hunter_web
+	multiplicative_slowdown = 0.2
 
 /datum/movespeed_modifier/duct_spider_web
 	multiplicative_slowdown = 1
 
+/datum/movespeed_modifier/nurse_web
+	multiplicative_slowdown = 1.2
+
+/datum/movespeed_modifier/tangle_web
+	multiplicative_slowdown = 1.2
+
+/datum/movespeed_modifier/midwife_web
+	multiplicative_slowdown = 1.2
+
+/datum/movespeed_modifier/ambush_web
+	multiplicative_slowdown = 5
+
+/datum/movespeed_modifier/tarantula_web
+	multiplicative_slowdown = 5
```

Comment:
If some of these have the same value, give them a more generic name and then just have one datum for that value. There's no reason to make multiple copies which do the same thing.

Reply:
> If some of these have the same value, give them a more generic name and then just have one datum for that value. There's no reason to make multiple copies which do the same thing.

Mostly did it differently so if someone wanted to balance a spider they would have an easier time but I have no problem with merging them into let's say very, slow,slow,avarage and fast categories